docs/source/examples/mitiq-paper/mitiq-paper-codeblocks.md

```diff
@@ -167,35 +167,29 @@ print("Original circuit:", circ, sep="\n")
 +++ {"id": "_5Q3P1DJfcar"}
 
 **Codeblock 9**
+<!-- #endregion -->
 
-```{code-cell} ipython3
----
-colab:
-  base_uri: https://localhost:8080/
-id: tSWVDcJIeuck
-outputId: e9d0c874-8814-4da0-c54b-a6881d3e614a
----
-folded = zne.scaling.fold_gates_from_left(
-    circ, scale_factor=2
-)
-print("Folded circuit:", folded, sep="\n")
+> Note: The method `zne.scaling.fold_gates_from_left` has been deprecated as of v0.36.0
```

Comment:
Two suggestions:

1. `deprecated` -> `removed`. I think deprecation implies a graceful error/warn which we are opting not to do. Do you agree?
2. We should use a warning [directive](https://myst-parser.readthedocs.io/en/latest/syntax/roles-and-directives.html) for this to make it pop out to the user a little more. Below is an example.

````md
```{warning}
The method `zne.scaling.fold_gates_from_left` has been removed as of v0.36.0.
```
````

Reply:
Agree with both suggestions.